src/HTML5Backend.js

```diff
@@ -113,7 +113,28 @@ export default class HTML5Backend {
     };
   }
 
-  connectDropTarget(targetId, node) {
+  adjustForDropTargetZoom(node, zoom) {
+    if (zoom) {
+      setZoom(zoom);
+    } else if (node.style && node.style.zoom) {
```

Comment:
Yes. Additionally, you can pass an explicit zoom value. That is how I'm using it in a project because the zoom attribute is specified on node higher up in the dom tree. When I get a chance, I'll play with getComputedStyle and currentStyle to see if I can get it to work off computed style.